### DISCUSSION TITLE
ask

### DISCUSSION THREAD

**Discussion by zlBaphomeT** [bangcorrupt/hacktribe#148] [General]
will it work with the e2 sampler ? :D

**Comment by zlBaphomeT** [comment]
and can I after the install of the hacktribe, use my on samples?

**Reply by maks** [reply]
yes and yes.

**Reply by zlBaphomeT** [reply]
Nice, I've already installed it and I'm very impressed. The OSC FX option on the sampler would be really awesome!
@bangcorrupt

**Reply by bangcorrupt** [reply]
Should be able to get the Osc Edit drive working at some point.  It exists in the sound engine and I made some progress communicating with the DSP to edit FX.

**Reply by zlBaphomeT** [reply]
![IMG-20220325-WA0047.jpg](https://user-images.githubusercontent.com/92951315/160111769-3fafa8f5-eb50-490b-b916-df3ab0249f09.jpg)
I mean that you can do something like the picture on the sampler. The picture is of the blue one from a friend

**Reply by bangcorrupt** [reply]
#77

**Reply by zlBaphomeT** [reply]
So it will be passable in the future ?
Sry my english is not so good 🙈

**Reply by bangcorrupt** [reply]
Yes, I'll add this in the future.

**Reply by zlBaphomeT** [reply]
> Yes, I'll add this in the future.

Thank you for this great work! It's a lot more fun to work with the electribe! 

I'm hyped for what's next for features!
Cuz the Blue Electribe 2 has more autenthic sounds and OSC sounds + Filters.

**Comment by soulsonique** [comment]
Can i restore from hacktribe to sampler 2.02 on my E2 grey synth? I upgraded to hacktribe from synth OS.

**Reply by bangcorrupt** [reply]
No, not yet.  You would need to copy the entire flash memory from a sampler to the synth.  This will be possible via sysex some day.
